pkg/config/types.go

```diff
@@ -95,6 +95,7 @@ const (
 	defaultKubeConfig      = ""
 	defaultBPFSampleRate   = 0
 	defaultCPUArchOverride = ""
+	defaultExcludeSwapperProcess = true
```

Comment:
This is a new option, let's make the default false first.

Reply:
updated, to change default behavior backward compatible.